cmd/snap-seccomp/versioninfo.go

```diff
@@ -65,3 +66,12 @@ func showVersionInfo() error {
 	fmt.Fprintln(os.Stdout, vi)
 	return nil
 }
+
+func GoSeccompFeatures() string {
+	features := "-"
+	// as more features are added, make this colon-separated
```

Comment:
Colon is a bit unusual, is there a specific reason for it?

Reply:
Well, passwd, group, shadow, gshadow beg to differ. The point is it shouldn't be a space. It doesn't matter otherwise.

Reply:
Indeed, I haven't thought about those. In either case I think this is fine, it's not a blocker from my side.